packages/victory-tooltip/src/flyout.js

```diff
@@ -99,6 +99,7 @@ Flyout.propTypes = {
   center: PropTypes.shape({ x: PropTypes.number, y: PropTypes.number }),
```

Comment:
Actually, what about adding `disableInlineStyles` to `CommonProps.primitiveProps` instead of in every component?

Reply:
I already have it in `primitiveProps` and `dataProps`, so I must have forgotten to remove it from this file.